test/SILOptimizer/Inputs/foo.h

```diff
@@ -0,0 +1,18 @@
+#ifndef VALIDATION_TEST_SILOPTIMIZER_FOO_H
```

Comment:
@zoecarver Can you change the name of this file to something more meaningful? I didn't notice the name when I was reviewing.